dist2src/cli.py

```diff
@@ -207,7 +215,9 @@ def convert(ctx, origin: str, dest: str):
         source_git_path=Path(dest_dir),
         log_level=ctx.obj[VERBOSE_KEY],
     )
-    if d2s.source_git_path.exists() and dest_branch in d2s.source_git.repo.branches:
+    if d2s.package_name in VERY_VERY_HARD_PACKAGES:
+        d2s.convert_single_commit(origin_branch, dest_branch)
```

Comment:
Why we have both this and a dedicated subcommand? Is this useful for something?

Reply:
whoops, good point! I thought I removed it /o\